docs/bot-usage.md

```diff
@@ -3,7 +3,7 @@
 This page explains the different parameters of the bot and how to run it.
 
 !!! Note
-    If you've used `setup.sh`, don't forget to activate your virtual environment (`source .env/bin/activate`) before running freqtrade commands.
+If you've used `setup.sh`, don't forget to activate your virtual environment (`source .env/bin/activate`) before running freqtrade commands.
```

Comment:
Please don't do this (it's not just in this file - but also in the other doc-files changed here).

after every line with `!!! Note ` or `!!! Bug` (or a few others) - the next line MUST be indented by 4 spaces.
It's required to have the boxes in the [documentation](https://www.freqtrade.io/en/latest/bot-usage/) render correctly.
It's alsy very picky to not have `!!! Note:` (no trailing colon)... but i think it looks good in the docs so it's worth the effort.

for further reference about this - see the [developer docs](https://www.freqtrade.io/en/latest/developer/#documentation) - which however only link to the original docs of the plugin we use for this [here](https://squidfunk.github.io/mkdocs-material/extensions/admonition/).

Reply:
Oh, I did not notice it auto formatted those. My bad, just fixed it, and fixed my editor settings for this repo.